isthisstockgood/DataFetcher.py

```diff
@@ -200,25 +201,38 @@ def parse_msn_money_annual_statement_data(self, response, *args, **kwargs):
     result = response.text
     self.msn_money.parse_annual_report_data(result)
 
-  def fetch_yahoo_finance_analysis(self):
```

Comment:
I think I slightly prefer the other naming. While in practice we only really parse the future growth rates. There could be a world where this fetches more data from the yahoo finance analysis payload.

It also then is a bit more clear

What do you think?

Reply:
Applied!